cookbooks/aws-parallelcluster-awsbatch/recipes/install.rb

```diff
@@ -28,9 +29,6 @@
 
 # Check whether install a custom aws-parallelcluster-awsbatch-cli package or the standard one
 # Install awsbatch cli into awsbatch virtual env
-if aws_region.start_with?("us-iso") && !node['cluster']['custom_awsbatchcli_package'].empty?
-  node.default['cluster']['custom_awsbatchcli_package'] = "#{node['cluster']['artifacts_s3_url']}/dependencies/awsbatch/aws-parallelcluster.tgz"
-end
```

Comment:
Is only removing the code enough? In the below code, it will try to install from pip and will fail, right

Reply:
From what I know we dont even support Batch scheduler in ADC

Reply:
it skips the whole install part for aws batch in ADC due to this line: `return if aws_region.start_with?("us-iso")`